python/src/sdf/pyLidar.cc

```diff
@@ -0,0 +1,109 @@
+         "sweep.")
+    .def("horizontal_scan_resolution", &sdf::Lidar::HorizontalScanResolution,
+         "Get the resolution for horizontal scan.")
+    .def("set_horizontal_scan_resolution", &sdf::Lidar::SetHorizontalScanResolution,
+         "Set the resolution for horizontal scan.")
+    .def("horizontal_scan_min_angle", &sdf::Lidar::HorizontalScanMinAngle,
+         "Get the minimum angle for horizontal scan.")
+    .def("set_horizontal_scan_min_angle", &sdf::Lidar::SetHorizontalScanMinAngle,
+         "Set the minimum angle for horizontal scan.")
+    .def("horizontal_scan_max_angle", &sdf::Lidar::HorizontalScanMaxAngle,
+         "Get the maximum angle for horizontal scan.")
+    .def("set_horizontal_scan_max_angle", &sdf::Lidar::SetHorizontalScanMaxAngle,
+         "Set the maximum angle for horizontal scan.")
+    .def("vertical_scan_samples", &sdf::Lidar::VerticalScanSamples,
+         "Get the number of lidar rays vertically to generate per laser "
+         "sweep.")
+    .def("set_vertical_scan_samples", &sdf::Lidar::SetVerticalScanSamples,
+         "Set the number of lidar rays vertically to generate per laser "
+         "sweep.")
+    .def("vertical_scan_resolution", &sdf::Lidar::VerticalScanResolution,
+         "Get the resolution for vertical scan.")
+    .def("set_vertical_scan_resolution", &sdf::Lidar::SetVerticalScanResolution,
+         "Set the resolution for vertical scan.")
+    .def("vertical_scan_min_angle", &sdf::Lidar::VerticalScanMinAngle,
+         "Get the minimum angle for vertical scan.")
+    .def("set_vertical_scan_min_angle", &sdf::Lidar::SetVerticalScanMinAngle,
+         "Set the minimum angle for vertical scan.")
+    .def("vertical_scan_max_angle", &sdf::Lidar::VerticalScanMaxAngle,
+         "Get the maximum angle for vertical scan.")
+    .def("set_vertical_scan_max_angle",
+         &sdf::Lidar::SetVerticalScanMaxAngle,
+         "Set the maximum angle for vertical scan.")
+    .def("range_min",
+         &sdf::Lidar::RangeMin,
+         "Get minimum distance for each lidar ray.")
+    .def("set_range_min", &sdf::Lidar::SetRangeMin,
+         "Set minimum distance for each lidar ray.")
+    .def("range_max",
+         &sdf::Lidar::RangeMax,
+         "Get the maximum angle for vertical scan.")
+    .def("set_range_max",
+         &sdf::Lidar::SetRangeMax,
+         "Set the maximum angle for vertical scan.")
+    .def("range_resolution",
+         &sdf::Lidar::RangeResolution,
+         "Get linear resolution of each lidar ray.")
+    .def("set_range_resolution",
+         &sdf::Lidar::SetRangeResolution,
+         "Set linear resolution of each lidar ray.")
+    .def("lidar_noise",
+         &sdf::Lidar::LidarNoise,
+         "Get the noise values for the lidar sensor.")
+    .def("set_lidar_noise", &sdf::Lidar::SetLidarNoise,
+         "Set the noise values for the lidar sensor.")
+    .def("__copy__", [](const sdf::Lidar &self) {
```

Comment:
I think `VisibilityMask` was added recently. Mind including it here?

Reply:
https://github.com/gazebosim/sdformat/pull/973/commits/58c6a26a4d0a3eb5f9949d994b43387c33574790